sdk/metricsadvisor/ai-metrics-advisor/test/public/datafeed.spec.ts

```diff
@@ -411,7 +411,7 @@ matrix([[true, false]] as const, async (useAad) => {
           assert.ok(result.value.status, "Expecting second data feed");
         });
 
-        it("lists datafeed by pages", async function() {
+        it.skip("lists datafeed by pages", async function() {
```

Comment:
Please make sure it's tracked by an issue to unskip this.

Reply:
https://github.com/Azure/azure-sdk-for-js/issues/15084 Issue tracking this